### PR TITLE
Complete user/transaction entities, add abstract base entity, rename models/ to entity/

### PR DESCRIPTION
**Pull request by sahamanish99**

### CODE REVIEW

File: src/entity/Base.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+abstract class Base {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ unique: true })
+  customerId: string;
```
Comment (young-k) [inline src/entity/Base.ts]: probably should have `createdAt` and `updatedAt` fields, since those are standard

Comment (sahamanish99) [reply src/entity/Base.ts]: I think @kq-li handled that in his branch, so it should be good to go when merged

Comment (sahamanish99) [reply src/entity/Base.ts]: also, it seems like naming convention is `/entity/*.ts` for typeorm

Comment (sahamanish99) [reply src/entity/Base.ts]: https://github.com/typeorm/typeorm/blob/master/README.md

Comment (young-k) [inline src/entity/Base.ts]: why is `customerId` in `Base`?

Comment (sahamanish99) [reply src/entity/Base.ts]: it's shared in both `User` and `Transaction`, so I moved it into `Base`. Should I leave `customerId` in `User` and `Transaction` separately?

Comment (young-k) [reply src/entity/Base.ts]: yep, because there may be more models in the future

**Comment by kq-li** [comment]
combined with #31

**Pull request closed without merging**